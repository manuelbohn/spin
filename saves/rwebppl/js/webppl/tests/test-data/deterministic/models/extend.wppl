var x = { a: 1, b: 2 };
var y = { b: 3, c: 4 };
var z = { d: 5 };
var w = extend(x, y, z);
[x.a, x.b, x.c || null, x.d || null, y.a || null, y.b, y.c, y.d || null, w.a, w.b, w.c, w.d];
