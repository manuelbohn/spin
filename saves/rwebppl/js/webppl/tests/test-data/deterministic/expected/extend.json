{
  "result": [1, 2, null, null, null, 3, 4, null, 1, 3, 4, 5]
}
